.github/workflows/test-suite.yml

```diff
@@ -21,13 +21,25 @@ jobs:
       - uses: "actions/setup-python@v2"
         with:
           python-version: "${{ matrix.python-version }}"
+
+      - uses: actions/cache@v2.1.6
+        id: cache-requirements
+        with:
+          path: ${{ env.pythonLocation }}
+          key: ${{ env.pythonLocation }}-${{ matrix.python-version }}-${{ hashFiles('requirements.txt', 'setup.py') }}
```

Comment:
Where have you gotten `${{ env.pythonLocation }}` from?
(I mean, it looks *neat*, but I don't understand it)

I don't see anything about it [mentioned in the docs, here].(https://github.com/actions/cache/blob/main/examples.md#python---pip)

Reply:
I took it from [this tutorial](https://medium.com/ai2-blog/python-caching-in-github-actions-e9452698e98d).

But it's defined on the `setup-action` above. Here specifically: https://github.com/actions/setup-python/blob/dc73133d4da04e56a135ae2246682783cc7c7cb6/src/find-python.ts#L62